fletcher/base.py

```diff
@@ -82,14 +81,24 @@ def __getitem__(self, item):
             start = item.start or 0
             stop = item.stop if item.stop is not None else len(self.data)
             stop = min(stop, len(self.data))
+            # Arrow can't handle slices with steps other than 1
```

Comment:
Please create an ARROW issue for that and mention it in the comment

Reply:
done